bwproject.py

```diff
@@ -164,16 +168,19 @@ def bare_request(self, verb, address_root, address_suffix, access_token="", para
             The response json
         """
         time.sleep(.5)
-        if access_token:
-            params["access_token"] = access_token
 
+        headers = {}
+
+        if access_token:
+            headers["Authorization"]="Bearer {}".format(access_token)
```

Comment:
Extremely not a big deal, but could use some white-space around the `=` to follow the standard; here and above